chanacceptor/interface.go

```diff
@@ -158,15 +158,21 @@ type ChannelAcceptResponse struct {
 	// channel was accepted, this value will be nil.
 	ChanAcceptError
 
+	// FundingAmount is the number of satoshis we will contribute to the
```

Comment:
I am not quite sure why we are adding the amount here, although the rpc is not changed in this PR, does it not make sense to include both in one PR?